### PR TITLE
Feature/user-logout

### PR DESCRIPTION
**Pull request by junior1384**

### CODE REVIEW

File: src/stores/user.ts [hunk 1]
```diff
@@ -15,7 +16,7 @@ const StateUser: UserStorage = {
 export const useUsers = defineStore(id, {
   state: () => {
     return {
-      StateUser,
+      StateUser: { ...UserStorage },
```
Comment (guigrasel) [inline src/stores/user.ts]: stateUser

File: src/stores/user.ts [hunk 2]
```diff
@@ -1,10 +1,11 @@
 import { defineStore } from "pinia";
 import GetUser from "../graphql/user/queries.gql";
-import { User, UserStorage } from "../entities/login";
+import { User } from "../entities/login";
+import { router } from "../modules";
 
 const id = "users";
 
-const StateUser: UserStorage = {
+const UserStorage = {
```
Comment (guigrasel) [inline src/stores/user.ts]: userStorage

File: src/stores/user.ts [hunk 3]
```diff
@@ -28,6 +29,11 @@ export const useUsers = defineStore(id, {
         email: form.labelEmail,
       });
     },
+    logout: () => {
+      const user = useUsers();
+      user.StateUser = UserStorage;
```
Comment (guigrasel) [inline src/stores/user.ts]:
```suggestion
      user.stateUser = userStorage;
```

File: src/stores/user.ts (hunk 2 above)
Comment (junior1384) [reply src/stores/user.ts]: Resolvido

File: src/stores/user.ts (hunk 1 above)
Comment (junior1384) [reply src/stores/user.ts]: Resolvido

File: src/stores/user.ts (hunk 3 above)
Comment (junior1384) [reply src/stores/user.ts]: Resolvido

**Review by guigrasel — APPROVED** [review]

**Pull request merged**